release-notes.md

```diff
@@ -150,14 +152,16 @@ The following updates were made in this release:
 
 The following updates were made in this release:
 
-**Android Core 1.4.4**
+**Android Core 1.4.4 (sdk-core:1.4.5)**
 
 * Starting in API level 16, notifications now support `BigTextStyle`.
 
   This enables long notifications to be displayed without being truncated.
 
 * Fixed the locale string in the HTTP User-Agent to follow the BCP 47 specification.
 
+These changes were released as part of `sdk-core` v1.4.5.
```

Comment:
- Please replace "(sdk-core: 1.4..5)" with "(sdk-core version 1.4.5)"

- Same question as the previous section...we already state that it's `sdk-core version 1.4..5"; do we need to state this again after the bullets?